src/containers/TriggerTemplate/TriggerTemplate.js

```diff
@@ -0,0 +1,386 @@
+        formattedLabelsToRender.push(`  ${splitOnColon[0]}: ${splitOnColon[1]}
+`);
+      });
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'default',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.default',
+          defaultMessage: 'Default'
+        })
+      },
+      {
+        key: 'description',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerTemplate.description',
+          defaultMessage: 'Description'
+        })
+      }
+    ];
+
+    let rowsForParameters = [];
+
+    if (params) {
+      rowsForParameters = params.map(
+        ({ name, default: defaultValue, description }) => ({
+          id: name,
+          name,
+          default: defaultValue,
+          description
+        })
+      );
+    }
+    return (
+      <>
+        <h1>{triggerTemplateName}</h1>
+        <Tabs>
+          <Tab
+            kind="ghost"
+            label={intl.formatMessage({
+              id: 'dashboard.triggerTemplate.details',
+              defaultMessage: 'Details'
+            })}
+          >
+            <div className="textarea">
+              <p className="triggertemplatetext">
+                <p>
+                  {intl.formatMessage({
+                    id: 'dashboard.triggerTemplate.creationTime',
+                    defaultMessage: 'Creation time: '
+                  })}
+                  <FormattedDate date={triggerTemplateDate} />
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    id: 'dashboard.triggerTemplate.namespace',
+                    defaultMessage: 'Namespace: '
+                  })}
+                  {triggerTemplateNamespace}
+                </p>
+
+                {triggerTemplateLabels && (
+                  <p>
+                    {intl.formatMessage({
+                      id: 'dashboard.triggerTemplate.labels',
+                      defaultMessage: 'Labels: '
+                    })}
+                    {formattedLabelsToRender.map(label => (
+                      <Tag type="blue">{label}</Tag>
+                    ))}
+                  </p>
+                )}
+              </p>
+            </div>
+          </Tab>
+          <Tab kind="ghost" label="YAML">
+            <div className="textarea">
+              <pre id="yaml">
+                <p className="triggertemplatetext">
+                  {jsYaml.dump(triggerTemplate)}
+                </p>
+              </pre>
+            </div>
+          </Tab>
+        </Tabs>
+
+        <DashboardTable
+          title="Parameters"
+          headers={headersForParameters}
+          rows={rowsForParameters}
+          selectedNamespace={selectedNamespace}
+          emptyTextAllNamespaces={intl.formatMessage(
```

Comment:
based on Alan's comment in my PR:
https://github.com/tektoncd/dashboard/pull/797#discussion_r353822648